src/schema/notifications.ts

```diff
@@ -41,6 +41,34 @@ function findDataId(
 	)[0];
 }
 
+function addNewContactGroupToList(array: any, item: any, sortBy: any) {
+	if (sortBy === 'nameAsc' && array.length > 0) {
```

Comment:
array is a common javascript keyword, please don't use as variable name, can confuse any new developer

Reply:
Updated